iris_grib/__init__.py

```diff
@@ -698,6 +694,10 @@ def _message_values(grib_message, shape):
     data = gribapi.grib_get_double_array(grib_message, 'values')
     data = data.reshape(shape)
 
+    # Handle missing values in a sensible way.
```

Comment:
Is this a hangover from not having masks?

Reply:
Not totally sure.
This is simply reflecting what is currently in Iris master.

Reply:
I'd love to find out more...